### PR TITLE
Fix issue where keyfile was not found by signing util

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Internal

### CODE REVIEW

**Comment by amshamah419** [comment]
Please note @IkaDemisto @yuvalbenshalom I'm running a test build first to verify

File: .circleci/config.yml [hunk 1]
```diff
@@ -523,9 +518,8 @@ jobs:
             PACK_ARTIFACTS=$CIRCLE_ARTIFACTS/content_packs.zip
             GCS_PATH=$(mktemp)
             echo $GCS_MARKET_KEY > $GCS_PATH
-            python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -b 'marketplace-dist' -s $GCS_PATH -n $CIRCLE_BUILD_NUM -p 'All' -pb 'marketplace-dist-private' -o
-            rm $GCS_PATH
-          when: on_success
+            python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -b 'marketplace-dist' -s $GCS_PATH -n $CIRCLE_BUILD_NUM -k $PACK_SIGNING_KEY -p 'All' -pb 'marketplace-dist-private' -o            rm $GCS_PATH
+          when: always
```
Comment (liorblob) [inline .circleci/config.yml:522]: Looks like this file should be reverted?

File: Tests/Marketplace/upload_packs.py [hunk 2]
```diff
@@ -399,7 +399,6 @@ def print_packs_summary(packs_list):
         print_error(f"Number of failed packs: {len(failed_packs)}")
         failed_packs_table = _build_summary_table(failed_packs)
         print_error(failed_packs_table)
-        sys.exit(1)
```
Comment (liorblob) [inline Tests/Marketplace/upload_packs.py:402]: intentional?

Comment (amshamah419) [reply Tests/Marketplace/upload_packs.py:402]: I re-added it from Ika's PR. - https://github.com/demisto/content/pull/6647

If we don't need it, I'll remove it. I more or less reverted everything.

Comment (IkaDemisto) [reply Tests/Marketplace/upload_packs.py:402]: @amshamah419 leave `sys.exit(1)`. From now, we want to fail the build if there are failed in upload packs

Comment (amshamah419) [reply Tests/Marketplace/upload_packs.py:402]: Makes sense. I'll re-add it

File: .circleci/config.yml (hunk 1 above)
Comment (IkaDemisto) [reply .circleci/config.yml:528]: @amshamah419 should be ` when: on_success` when you finish testing

**Pull request merged**